network/topology/randomizedTopology.go

```diff
@@ -65,6 +72,7 @@ func (r RandomizedTopology) GenerateFanout(ids flow.IdentityList) (flow.Identity
 		// no subscribed channel id, hence skip topology creation
 		// we do not return an error at this state as invocation of MakeTopology may happen before
 		// node subscribing to all its channels.
+		r.logger.Warn().Msg("skips generating fanout with no subscribed channels")
```

Comment:
except for node startup, this is an indicator of topology disconnection on the network.